public/app/features/serviceaccounts/state/actions.ts

```diff
@@ -5,15 +5,15 @@ import { serviceAccountsLoaded } from './reducers';
 
 export function loadServiceAccounts(): ThunkResult<void> {
   return async (dispatch) => {
-    const serviceAccounts = await getBackendSrv().get('/api/serviceaccounts');
+    const serviceAccounts = await getBackendSrv().get('/api/org/serviceaccounts');
```

Comment:
To make this more robust in case of future changes, you can save the base url to a variable and use that in the routes:
```
const BASE_URL = `'/api/org/serviceaccounts'`;
...
const serviceAccounts = await getBackendSrv().get(BASE_URL);
...
await getBackendSrv().patch(`{BASE_URL}/${serviceAccount.serviceAccountId}`, {
      role: serviceAccount.role,
    });

Reply:
let's do it!